src/components/PageNavigator.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import clsx from 'clsx';
+
+import Button from './Button';
+import SimpleSelect from './SimpleSelect';
+import Icon, { IconInputType } from './Icon';
+
+type PageNavigator = {
+  totalCount: number,
+  itemsPerPage: number,
+  setItemsPerPage: (count: number) => void,
+  hideItemsPerPage?: boolean
+  showControls?: boolean,
+  showEdges?: boolean,
+  page: number,
+  setPage: (no: number) => void,
+  id?: string,
+  className?: string,
+  icons?: {
+    next?: IconInputType,
+    prev?: IconInputType,
+    start?: IconInputType,
+    end?: IconInputType,
+  }
```

Comment:
no need for the user to customize the icons, assume the default

Reply:
done 👍🏽